tests/components/wyoming/test_conversation.py

```diff
@@ -0,0 +1,239 @@
+"""Test conversation."""
+
+from __future__ import annotations
+
+from unittest.mock import patch
+
+from syrupy import SnapshotAssertion
+from wyoming.asr import Transcript
+from wyoming.handle import Handled, NotHandled
+from wyoming.intent import Entity, Intent, NotRecognized
+
+from homeassistant.components import conversation
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import Context, HomeAssistant
+from homeassistant.helpers import intent
+
+from . import MockAsyncTcpClient
+
+
+async def test_intent(hass: HomeAssistant, init_wyoming_intent: ConfigEntry) -> None:
+    """Test when an intent is recognized."""
+    agent = conversation.async_get_agent(hass, init_wyoming_intent.entry_id)
```

Comment:
This is wrong. You need to do lookups based on entity IDs. Entry ID should not be used.